packages/kit/src/core/dev/index.js

```diff
@@ -346,7 +346,7 @@ async function create_handler(vite, config, dir, cwd, manifest) {
 						hooks: {
 							getSession: hooks.getSession || (() => ({})),
 							handle: hooks.handle || (({ request, resolve }) => resolve(request)),
-							serverFetch: hooks.serverFetch || fetch
+							externalFetch: hooks.externalFetch || fetch
```

Comment:
copying a comment over from the old PR

on line ~293 we have a check for old hooks. we should add a check for `serverFetch` as well

```
				if (/** @type {any} */ (hooks).getContext) {
					// TODO remove this for 1.0
					throw new Error(
						'The getContext hook has been removed. See https://kit.svelte.dev/docs#hooks'
					);
				}
```

Reply:
```js
if (/** @type {any} */ (serverFetch).getContext) {
					// TODO remove this for 1.0
					throw new Error(
						'The serverFetch was renamed to externalFetch. See https://kit.svelte.dev/docs#hooks-externalfetch'
					);
				}
```

so something like this?

Reply:
I tried, but I'm totally lost in this. I don't know how to define value of if condition.

Reply:
I would think that on line 300, you can add:
```
				if (/** @type {any} */ (hooks).serverFetch) {
					// TODO remove this for 1.0
					throw new Error(
						'The serverFetch hook has been renamed to externalFetch'
					);
				}
```

Reply:
oh, ofc... I'm dumb.